### PR TITLE
Update Selenium Level Sponsor (Experitest becomes Digital.ai)

### PR DESCRIPTION
**Pull request by bonigarcia**
### Description
This PR updates the name of a Selenium level sponsor: Experitest is now Digital.ai.

### Motivation and Context
I recently received the following email (CC @manoj9788, @mmerrell, @pujagani) from a Product Manager from Digital.ai:

_Dear Selenium Project contributors and project leadership team,_

_In 2020, the Selenium Level Sponsor [Experitest was acquired by Digital.ai](https://www.businesswire.com/news/home/20200616005084/en/Digital.ai-Acquires-Numerify-and-Experitest-%E2%80%93-Accelerating-Journey-to-Become-the-Industry%E2%80%99s-Intelligent-End-to-End-Value-Stream-Management-Solution), an intelligent Value Stream Platform that provides a continuous testing solution. If you could kindly update the footer of selenium.dev with the new logo (attached), that would be very much appreciated._

_Thank you for your continued work on this important project,_

### Types of changes

- [X] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [X] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: website_and_docs/data/sponsors.yml [hunk 1]
```diff
@@ -10,10 +10,10 @@ selenium:
     - logo: "/images/sponsors/saucelabs.png"
       url: "http://saucelabs.com"
       name: "Sauce Labs"
-    # Sponsorship start date: June 24, 2015
-    - logo: "/images/sponsors/experitest.png"
+    # Sponsorship start date: June 24, 2015, updated July 2023
```
Comment (harsha509) [inline website_and_docs/data/sponsors.yml:13]: Can we have start date too in here (or) can we just by Month wise ?

Any thoughts on this @diemol !

**Review by Krafman — APPROVED** [review]
Hi all  (@harsha509  @diemol  et al)

The image looks great and we are looking forward to seeing the change implemented.

File: website_and_docs/data/sponsors.yml (hunk 1 above)
Comment (diemol) [reply website_and_docs/data/sponsors.yml:13]: This is a @SeleniumHQ/selenium-plc matter. So not sure. However, they said we could merge the PR.

**Comment by diemol** [comment]
@SeleniumHQ/selenium-plc said we could merge the PR.

**Pull request merged**